core-plugins/src/test/java/co/cask/hydrator/plugin/batch/BatchPluginsTestSuite.java

```diff
@@ -25,7 +26,6 @@
  */
 @RunWith(Suite.class)
 @Suite.SuiteClasses({
-  BatchCubeSinkTestRun.class,
```

Comment:
i ran the package tests in IDE, but have removed the 'Run' suffix to keep the standard.